src/models/fhir/required.test.ts

```diff
@@ -1,5 +1,6 @@
 import { R4 } from "@ahryman40k/ts-fhir-types";
 import { pdtHealthCertV2 } from "@govtechsg/oa-schemata";
+import { cloneDeep } from "lodash";
 import { DetailedCodedError } from "../../common/error";
 import fhirHelper from "./index";
 import exampleHealthCertWithNric from "../../../test/fixtures/v2/pdt_pcr_with_nric_unwrapped.json";
```

Comment:
Could help to take this chance to rename as well? Thanks! 🙏🏻

```suggestion
import examplePcrHealthCertWithNric from "../../../test/fixtures/v2/pdt_pcr_with_nric_unwrapped.json";
```